keyboards/system76/launch_1/readme.md

```diff
@@ -0,0 +1,99 @@
+
+- High-speed USB Hub
+- Works on Linux, Windows and macOS
+- 100% Open Source
+- Made in Colorado
+
+Additional Launch Keyboard resources:
+
+- Keyboard Maintainer: [System76](https://github.com/system76)
+- Hardware Supported: [System76 Launch GitHub Repository](https://github.com/system76/launch)
+- Hardware Availability: [Shop System76](https://system76.com/accessories/launch)
+
+## Prerequisites
+
+Refer to [build environment setup](https://beta.docs.qmk.fm/tutorial/newbs_getting_started) instructions.
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+- **Bootmagic reset**: Hold down the key at (0,0) in the matrix (i.e., Escape) and plug in the keyboard
+- **Keycode in layout**: Press the key mapped to `RESET` if it is available
+<!-- - **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead -->
+
+## Building Firmware
+
+Start with QMK's [Complete Newbs Guide](https://beta.docs.qmk.fm/tutorial/newbs).
+
+In summary:
+
+- Clone this repository and `cd` into the `qmk_firmware` directory.
+- After cloning, run `make git-submodule` as well as `./util/qmk_install`, if required.
+- May also need to install dependencies, for example in Debian-based systems: `sudo apt install avrdude gcc-avr avr-libc`
+- To build the firmware without flashing the keyboard, use `make (keyboard name):(layout name)`; see the [`make` instructions](https://beta.docs.qmk.fm/developing-qmk/qmk-reference/getting_started_make_guide) for more details.
+- For example, to build the `default` layout for the Launch keyboard using `make`, run:
+
+```bash
+make -r system76/launch_1:default
+```
+
+- Equivalently, using the QMK CLI:
+
+```bash
+qmk compile -kb system76/launch_1 -km default
+```
+
+## Flashing Firmware (DFU)
+
+To flash the firmware, use the same build command, but with `dfu` added to the end:
+
+```bash
+make -r system76/launch_1:default:dfu
+```
+
+- After it builds, there will be repeating message that says:
+
+```console
+dfu-programmer: no device present.
+ERROR: Bootloader not found. Trying again in 5s.
+```
+
+Next, unplug the keyboard from the machine, hold down the Escape key (while the keyboard is unplugged), and plug the keyboard back in while still holding the Escape key down. Once the keyboard is plugged in, the Escape key can be released.
+
+> Note: On some distributions, `dfu-programmer` has trouble detecting the keyboard unless `make` is run through `sudo`.
+
+## Flashing Firmware (ISP)
+
+To flash the firmware using ISP, use a USBasp device and a tag connect cable.
+
+Build the firmware and bootloader with:
+
+```bash
+make -r system76/launch_1:default:production
+```
+
+Run `avrdude` to flash the fuses:
+
+```bash
+avrdude -c usbasp -p atmega32u4 -U lfuse:w:0x5E:m -U hfuse:w:0xD9:m -U efuse:w:0xCB:m
+```
+
+Run `avrdude` to flash the ROM:
+
+```bash
+avrdude -c usbasp -p atmega32u4 -U flash:w:system76_launch_1_default_production.hex
+```
```

Comment:
```suggestion
```
This process is already documented in the ISP flashing guide.

Reply:
Agreed and I have linked to it. However, the guide's information is scattered, mixed in with a variety of unrelated hardware, the steps are out-of-order, and the specific fuse settings for this board's configuration are not documented nor are they easily derivable unless one reads the schematics. Furthermore, this section is in the original documentation that System76 provides in their fork repo and I am hesitant to remove it.

Reply:
The steps are definitely not out of order. Are you looking at an old version? I have recently updated this page.

Reply:
OK.